daphne_worker/src/dap.rs

```diff
@@ -589,6 +589,10 @@ where
         try_join_all(requests).await.map_err(dap_err)?;
         Ok(())
     }
+
+    async fn current_batch(&self, task_id: &Id) -> std::result::Result<Id, DapError> {
+        self.internal_current_batch(task_id).await
```

Comment:
Do you think that we still want `internal_current_batch()`?

Reply:
I did this because it means I can leave the config stuff in config, and the trait implementation can call into it.